CHANGES.rst

```diff
@@ -5,13 +5,13 @@
 New Features
 ^^^^^^^^^^^^
 
+- Add keywords.deleter method to ImageFileCollection [#474]
 
```

Comment:
Could you add a blank line after the item here? Makes it easier to distinguish sections.